assets/bem/blocks/_book.scss

```diff
@@ -0,0 +1,58 @@
+.book {
+    &--120 {
+        width: 120px;
+    }
+    // &--mr {
+    //     margin-right: 24px;
+    // }
+    &__cover-img {
+        width: 120px;
+        height: 176px;
+    }
+    &__description {
```

Comment:
While doing BEM in this fashion is technically possible I find it super painful. Once app grows developers will be tasked with changing something like the description of a book. They go to the inspector and they see `<div class="book_description">` simple so they go to the code editor and search for "book_description" but get zero results because no where in css does the string "book_description" exist. 😦 So then they have to try and figure out what sort of string concatenation trickery is happening with the class name.

Not saying you have to change it. If you like it I'm fine with it, just sharing my experience.

```
.book__cover-img {
...
}

.book__description {
...
}
```

Reply:
Ok, thank you, I will think about it and will try to find another way

Reply:
Which practice do you recommend instead of BEM?